encoding/legacy-mb-tchinese/big5/big5-encode-form-errors-extBa.html

```diff
@@ -4,10 +4,33 @@
 <meta charset="big5"> <!-- test breaks if the server overrides this -->
 <title>Big5 encoding errors (form, extB, part1)</title>
 <meta name="timeout" content="long">
+<meta name="variant" content="?1-1000">
+<meta name="variant" content="?1001-2000">
+<meta name="variant" content="?2001-3000">
+<meta name="variant" content="?3001-4000">
+<meta name="variant" content="?4001-5000">
+<meta name="variant" content="?5001-6000">
+<meta name="variant" content="?6001-7000">
+<meta name="variant" content="?7001-8000">
+<meta name="variant" content="?8001-9000">
+<meta name="variant" content="?9001-10000">
+<meta name="variant" content="?10001-11000">
+<meta name="variant" content="?11001-12000">
+<meta name="variant" content="?12001-13000">
+<meta name="variant" content="?13001-14000">
+<meta name="variant" content="?14001-15000">
+<meta name="variant" content="?15001-16000">
+<meta name="variant" content="?16001-17000">
+<meta name="variant" content="?17001-18000">
+<meta name="variant" content="?18001-19000">
+<meta name="variant" content="?19001-20000">
+<meta name="variant" content="?20001-21000">
+<meta name="variant" content="?21001-last">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="big5_index.js"></script>
 <script src="big5-encoder.js"></script>
+<script src="/common/subset-tests.js"></script>
```

Comment:
Just as a style thing, I'd expect /common/ scripts to be listed under /resources/ and before the module- or test-specific scripts.

Reply:
Utility code is usually in common I think. I can move the `<script>` though.

Reply:
Yeah, sorry, I just meant the script tag